packages/networks/tsconfig.json

```diff
@@ -0,0 +1,73 @@
+{
```

Comment:
I wonder if this should be shared at the root of the repo? wdyt?

Reply:
yep the best way is to have a `ts-config` package that we can import in the different workspaces. I think we keep it local for now then will add a package in another PR